db/data/goals.yml

```diff
@@ -0,0 +1,80 @@
+# List of Cool Climate goals
+# NOTE: API key = 'input_takeaction_' + id.
+- id: carpool
+  configuration:
+    - id: commute_car_type
+    - id: commute_car_age
+    - id: commute_car_efficiency
+  actions:
+    - id: commute
+      questions:
+        - id: commute_mode
+        - id: commute_distance
+        - id: commute_carpool
+          parent:
+            id: commute_mode
+            logic:
+              '==': car
```

Comment:
This is JSON logic: http://jsonlogic.com/

Basically it means we will compare the answer for `compute_mode` and then:

```js
if (jsonLogic.apply( { "==" : ['car', answers.commute_mode] } )){
  // show question commute_carpool
}
```